GooglePlayPlugins/com.google.play.billing/Samples/TrivialKart/scripts/StoreController.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class StoreController : MonoBehaviour
+{
+    public GameObject tab;
+    public GameObject gasPage;
+    public GameObject coinPage;
+    public GameObject carPage;
+    public Text coinsCount;
+
+    private GameObject[] _tabs;
+    private int _tabsCount;
+    
+    // Start is called before the first frame update
+    void Start()
+    {
+        _tabsCount = tab.transform.childCount;
+        _tabs = new GameObject[_tabsCount];
+        for (int i = 0; i < _tabsCount; i++)
+        {
+            _tabs[i] = tab.transform.GetChild(i).gameObject;
+        }
+    }
+
+    private void Update()
+    {
+        coinsCount.text = PlayerPrefs.GetInt("coins", 20).ToString();
```

Comment:
what is this 20?

Reply:
it is the default amount of coins if playerPref doesn't have any record.